api/v1alpha4/machinedeployment_types.go

```diff
@@ -28,6 +28,9 @@ const (
 	// i.e. gradually scale down the old MachineSet and scale up the new one.
 	RollingUpdateMachineDeploymentStrategyType MachineDeploymentStrategyType = "RollingUpdate"
 
+	// Replace old MachineSets when the deletion of the associated machines are completed.
```

Comment:
nit: comment should start with the name of the const

```suggestion
	// OnDeleteMachineDeploymentStrategyType replaces old MachineSets when the deletion of the associated machines are completed.
```